config/database.yml

```diff
@@ -1,5 +1,6 @@
 default: &default
   adapter: postgresql
+  encoding: utf8
```

Comment:
according to http://www.postgresql.org/docs/9.0/static/multibyte.html 'utf8' is an alias for 'unicode'

as other apps use 'unicode' we should be consistent 
@pixeltrix the rails default is unicode encoding for postgres?